# **Statistics Basic-2 Assignment:-**

# **Q1:-**
## **What are the three measures of central tendency?**

## **Ans:-**

### **There are three main measures of central tendency:**
### **1.mode.**
### **2.median**
### **3.mean**

# **Q2:-** 
## **What is the difference between the mean, median, and mode? How are they used to measure the central tendency of a dataset?**

## **Ans:-**

### **The mean is the average where the sum of all the numbers is divided by the total number of numbers, whereas the median is the middle value in the list of given numbers numerically ordered from smallest to biggest and mode is the value of the number which occurs most often in the list.**

# **Q3:-** 
## **Measure the three measures of central tendency for the given height data:**
### **[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]**

## **Ans:-**

In [64]:
import numpy as np

In [65]:
height=[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

###  **1.Mean:-**

In [66]:
np.mean(height)

177.01875

###  **2.Median:-**

In [67]:
np.median(height)

177.0

###  **3.Mode:-**

In [68]:
from scipy import stats

In [69]:
stats.mode(height)

ModeResult(mode=177.0, count=3)

# **Q4:-** 
## **Find the standard deviation for the given data:**
### **[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]**

## **Ans:-**

In [70]:
data=[178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

In [71]:
np.std(data)

1.7885814036548633

# **Q5:-** 
## **How are measures of dispersion such as range, variance, and standard deviation used to describe the spread of a dataset? Provide an example.**

## **Ans:-**

### **Two data sets can have the same mean but they can be entirely different. Thus to describe data, one needs to know the extent of variability. This is given by the measures of dispersion. Range, interquartile range, and standard deviation are the three commonly used measures of dispersion.**

# **Q6:-** 
## **What is a Venn diagram?**

## **Ans:-**

### **A diagram representing mathematical or logical sets pictorially as circles or closed curves within an enclosing rectangle (the universal set), common elements of the sets being represented by intersections of the circles.**

# **Q7:-** 
## **For the two given sets A = (2,3,4,5,6,7) & B = (0,2,6,8,10). Find:**
### **(i) A intersection B**
### **(ii) A ⋃ B**

## **Ans:-**

### **(i) A intersection B**
#### **(2,6)**

### **(ii) A ⋃ B**
#### **(0,2,3,4,5,6,7,8,10)**

In [72]:
A = {2,3,4,5,6,7}
B = {0,2,6,8,10}

In [73]:
z = A.union(B)
print("Union:-",z)

Union:- {0, 2, 3, 4, 5, 6, 7, 8, 10}


In [74]:
z = A.intersection(B)
print("Intersection:-",z)

Intersection:- {2, 6}


# **Q8:-** 
## **What do you understand about skewness in data?**

## **Ans:-**

### **Skewness is a measurement of the distortion of symmetrical distribution or asymmetry in a data set. Skewness is demonstrated on a bell curve when data points are not distributed symmetrically to the left and right sides of the median on a bell curve.**

# **Q9:-** 
## **If a data is right skewed then what will be the position of median with respect to mean?**

## **Ans:-**

### **For a right skewed distribution, the mean is typically greater than the median.**

# **Q10:-** 
## **Explain the difference between covariance and correlation. How are these measures used in statistical analysis?**

## **Ans:-**

### **Both covariance and correlation measure the relationship and the dependency between two variables. Covariance indicates the direction of the linear relationship between variables. Correlation measures both the strength and direction of the linear relationship between two variables. Correlation values are standardized.**

# **Q11:-** 
## **What is the formula for calculating the sample mean? Provide an example calculation for a dataset.**

## **Ans:-**

### **The general sample mean formula for calculating the sample mean is expressed as x̄ = ( Σ xi ) ÷ n. Here, x̄ denotes the average value of the samples or sample mean, xi refers all X sample values and 'n' stands for the number of sample terms in the given data.**

# **Q12:-** 
## **For a normal distribution data what is the relationship between its measure of central tendency?**

## **Ans:-**

### **In normal distribution, measure of central tendency is equal.**

# **Q13:-** 
## **How is covariance different from correlation?**

## **Ans:-**

### **Covariance indicates the direction of the linear relationship between variables. Correlation measures both the strength and direction of the linear relationship between two variables.**

# **Q14:-** 
## **How do outliers affect measures of central tendency and dispersion? Provide an example.**

## **Ans:-**

### **Outliers are numbers in a data set that are vastly larger or smaller than the other values in the set. Mean, median and mode are measures of central tendency. Mean is the only measure of central tendency that is always affected by an outlier. Mean, the average, is the most popular measure of central tendency.**

In [75]:
A = [1,2,3,4,5,6,7,8,9]
print("mean:-",np.mean(A))
print("median:-",np.median(A))

mean:- 5.0
median:- 5.0


In [76]:
A = [1,2,3,4,5,6,7,8,9,200]
print("mean:-",np.mean(A))
print("median:-",np.median(A))

mean:- 24.5
median:- 5.5


#### **Here we notice that the mean value have very large change 5.0 to 24.5 but median have not change as like to mean**